docs/0900|Sync_With_Database.md

```diff
@@ -0,0 +1,106 @@
+## Syncing to Database Recipie with Firebase 
+
```

Comment:
The file name maps to the name of this page on the docs site. I think we should call this page Recipes so please rename the file to `0900|Recipes.md`

Reply:
Changed the name to `0900|Recipes.md`!